components/molecules/imageLocation/components/imageLocationFormElement.js

```diff
@@ -0,0 +1,51 @@
+/**
+ * Image Location FormElement
+ */
+
+// React
+import React from 'react'
+import { func, object } from 'prop-types'
+import { Controller } from 'react-hook-form'
+
+// Style
+import styled, { keyframes } from 'styled-components'
+
+// UI
+import { ImageLocation } from './imageLocation'
+import { ImageLocationProps } from './props'
+const elementName = 'imageLocationData'
+export const ImageLocationFormElement = ({ control, errors, setValue, ...props }) => {
+  return (
+    <ControllerWrapper className={`${errors && errors[elementName] && 'hasError'}`}>
+      <Controller
+        {...props}
+        as={ImageLocation}
+        control={control}
+        coordinatesChange={imageLocationData => {
+          setValue('imageLocationData', imageLocationData)
+        }}
+        name={elementName}
+        rules={{ required: true }}
```

Comment:
@mhsnprvr2 We do not used required: true on any form elements - plan is to ONLY use yup for validation, this is opinionated but working well right now.